skipper.go

```diff
@@ -1456,6 +1459,10 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 		)
 	}
 
+	if len(o.CompressEncodings) > 0 {
+		o.CustomFilters = append(o.CustomFilters, builtin.NewCompressWithEncodings(o.CompressEncodings))
```

Comment:
I think we need to remove `NewCompress()` from built-in registry to avoid name clash

Reply:
I think we should not remove it from `builtin.MakeRegistry()` as it could potentially break library users (but I think we've done this for some other filters in the past that required new config), @szuecs @aryszka WDYT?

Maybe its fine to put it into custom filters, we would need to update a comment below (and/or add a log to `registry.Register`) to hint that `CustomFilters` may override built-in.

Reply:
We could also move NewCompress() to the else branch of the code here and this should not break skipper library users.